docs/readmes/orc8r/dev_gateway_registration.md

```diff
@@ -0,0 +1,85 @@
+---
+id: dev_gateway_registration
+title: Gateway Registration
+hide_title: true
+---
+# Gateway Registration
+
+## Motivation
+
+Previously, to register a gateway, the operator would have to gather the gateway information (both the hardware id and the challenge key) through `show_gateway_info.py` script, then provision the gateway with the correct values of `control_proxy.yml` and `rootCA.pem`.
+Then, the operator would have to register each gateway through the NMS UI and validate the gateway registration by running `checkin_cli.py`.  
+
+In efforts to simplify gateway registration, the new registration process only requires two steps: registering the gateway at Orc8r and then running `register.py` at the gateway.  
+
+## Overview
+
+The overview of gateway registration is as follows:
+![gateway_registration_overview](assets/orc8r/gateway_registration_overview.png)
```

Comment:
The graphics uploaded above seems incomplete. I.e. the lines touching the bottom of the graphics indicate to me, that there is more to come in the process. Could we add a "termination" there indicating that the registration process is finished?

Reply:
Yeah I was using https://sequencediagram.org/ to create the diagram, it seems to format all its sequence diagrams like so

Reply:
What do you think about using the `bottomparticipants` keyword in that tool to somehow work around that issue?

Reply:
If you created the graphics from some text (similar to mermaid), than you would probably also want to add a hidden section with the respective code to the docs.
In Github this would probably be a `<!-- HMTL comment block -->`, maybe Docusaurus has something similar?

Reply:
If hidden parts are not possible, you might want to use [a details block](https://docusaurus.io/docs/markdown-features#details).

Reply:
Hi @christinewang5 ,
according to this [slack message](https://magmacore.slack.com/archives/C018J8UMGMR/p1649218904594389?thread_ts=1649157037.366159&cid=C018J8UMGMR) DocuSaurus does support mermaid diagrams. (Also see the examples [for Github](https://github.com/magma/magma/blob/master/docs/readmes/feg/s1ap_federated_test.md) and [Docusaurus](https://docs.magmacore.org/docs/next/feg/s1ap_federated_tests#s1ap-federated-integration-tests)).

Could you at least post the code, used to generate the graphics here in the thread, for a later cleanup, please.